apps/client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@beequant/client",
   "version": "0.1.0",
   "scripts": {
-    "dev": "concurrently 'yarn:dev-*' --restart-tries 3 --restart-after 3000",
+    "dev": "yarn dev-codegen & yarn dev-next",
```

Comment:
yeah, you need to fix these kind of changes before I can do reviews

Reply:
reset with dev branch